File: Logic/1 - C++/exercicio4.cpp
#include <stdio.h>
int main(void)
{
    float salario, reajuste;
    printf("Qual o salario do funcionario?");
    scanf("%f",&salario);
    printf("Qual a taxa de reajuste do salario?");
    scanf("%f", &reajuste);
    salario=salario*(1+(reajuste/100));
    printf("O salario será de R$%f com o rejuste anual de %f%% no primeiro ano.", salario, reajuste);
}

File: Exercicios e atividades/Exercicios/3 - JavaScript/exercicio4.js
var salario, reajuste;
    salario = prompt("Qual o salário do funcionário?");
    reajuste = prompt("Qual a taxa de reajuste do salário?");
    salario=salario*(1+(reajuste/100));
    alert("O salário será de R$" + salario + " com o rejuste anual de " + reajuste + "% no primeiro ano.");

